metabase/metabase
# Use recipients API instead of useUserListQuery

### Description

Changes the `SearchUserPicker` and `InfoText` components' calls:
From: `useUserListQuery` - `/api/user`
  To: `UserApi.list` - `/api/user/recipients`


For now, this will allow non-admin users to filter and see other users within the search app.


### How to verify

1. Create/access a user without admin permissions.
2. Go to the search app (i.e. type in a search and click the `View and filter results` button
3. Try and filter by `Creator` or `Last editor`
4. You should at least see an admin user, along with other users in your instance.

### Checklist
